frontend/README.md

```diff
@@ -0,0 +1,97 @@
+# Organics Info Frontend
+
+Refer to the project README file in the parent directory for details 
+on the architecture, libraries, and how to run the whole application.
+
+The information in this README file will be specific to the frontend only.
+
+## Scripts
+
+See the `scripts` section of the `frontend/package.json` file to see 
+what command line scripts can be run. The most used ones are:
+- `npm start` - Starts the local application
+- `npm run dev` - Starts the app with network support
+- `npm run lint` - Runs the eslint to check for linting issues
+- `npm run tsc` - Runs the typescript checker for typing issues
+- `npm run test:unit` - Runs unit tests in watch mode
+- `npm run test:cov` - Runs unit tests and generates coverage report
+- `npm run test-e2e` - Runs Playwright end to end tests
+
+There is also a `scripts` folder that contains other utility scripts 
+that can be run. See the README file in that directory for more details.
+
+## Tests - Playwright and Testing Library
+
+The [Playwright](https://playwright.dev/) library is used for End to End tests.
+It requires the backend server and front end client to be running.
+
+[React Testing Library](https://testing-library.com/) is used for unit tests, 
```

Comment:
we are using vitest for unit tests of frontend https://vitest.dev/

Reply:
Great catch, I forgot about that!